<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataCollector</class>
 <widget class="QMainWindow" name="DataCollector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DataCollector</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="Port_comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string>Invalid</string>
    </property>
    <item>
     <property name="text">
      <string>Invalid</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>/dev/ttyACM0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>/dev/ttyACM1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>/dev/ttyUSB0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>/dev/ttyUSB1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM10</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="Comm_label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/ledred.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="data_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>781</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="Scan_pushButton">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>60</y>
       <width>151</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Scan</string>
     </property>
    </widget>
    <widget class="QLabel" name="SCAN_label">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>60</y>
       <width>22</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../A_os/Qt/QtXmodem/ledgreen.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_SensorInfo">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>20</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_45">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>41</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Line</string>
     </property>
    </widget>
    <widget class="QComboBox" name="SensorInfoSensor_comboBox">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>All</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="GetSensorInfoCommand_pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Sensor Version</string>
     </property>
    </widget>
    <widget class="QComboBox" name="SensorInfoLine_comboBox">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>All</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_46">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Sensor</string>
     </property>
    </widget>
    <widget class="QComboBox" name="NumberOfSensors_comboBox">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="currentText">
      <string>8</string>
     </property>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_48">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of sensors</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Power_pushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Power ON</string>
    </property>
   </widget>
   <widget class="QFrame" name="Info_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>781</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="ConcentratorVersion_pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>191</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string> Concentrator Version</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_CONCENTRATORVERSION">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>421</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="SelectAlgoCSVFile_pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>191</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Algo CSV File   </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_CSVFILE">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>421</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="useK_checkBox">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>40</y>
       <width>131</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Use K table</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="DEBUG_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>781</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="QSENSOR_DEBUG_comboBox">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>40</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>All</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="DSCDebugEnable_checkBox">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable COM Debug Prints</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_42">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>DSC debug Print</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_43">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>40</y>
       <width>41</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Line</string>
     </property>
    </widget>
    <widget class="QComboBox" name="QLINE_DEBUG_comboBox">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>40</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>All</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_44">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>40</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Sensor</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CSVDebugEnable_checkBox">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>231</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable CSV Debug Prints</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_47">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>CSV debug Print</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="AlgoDebugEnable_checkBox">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>10</y>
       <width>241</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable ALGO Debug Prints</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
